VadeSecure/vade_secure_m365/ingest/parser.yml

```diff
@@ -0,0 +1,74 @@
+name: vadesecure-m365
+pipeline:
+-   external:
+        name: json.parse-json
+        properties:
+            input_field: '{{original.message}}'
+            output_field: message
+    name: parse_json
+-   name: categorization
+-   name: generic_email
+    filter: '{{parse_json.message.from != null}}'
+-   name: custom_vade
+-   name: vade_email
+    filter: '{{parse_json.message.from != null}}'
+-   name: vade_campaign
+    filter: '{{parse_json.message.from == null}}'
+stages:
+    categorization:
+        actions:
+        -   name: set
+            set:
+                event.category: email
+                event.kind: event
+                event.type: info
+        -   filter: '{{parse_json.message.action=="MOVE"}}'
+            name: set
+            set:
+                event.type: change
+        -   filter: '{{parse_json.message.action=="DELETE"}}'
+            name: set
+            set:
+                event.type: deletion
+        -   filter: '{{parse_json.message.action=="STRIP_ATTACHMENT"}}'
+            name: set
+            set:
+                event.action: dropped
+                event.type: denied
+    custom_vade:
+        actions:
+        -   name: set
+            set:
+                vadesecure.reason: '{{parse_json.message.reason}}'
+    vade_email:
+        actions:
+        -   name: set
+            set:
+                vadesecure.folder: '{{parse_json.message.folder}}'
+                vadesecure.from_header: '{{parse_json.message.from_header}}'
+                vadesecure.to_header: '{{parse_json.message.to_header}}'
+                vadesecure.status: '{{parse_json.message.status}}'
+                vadesecure.substatus: '{{parse_json.message.substatus}}'
+    vade_campaign:
+        actions:
+        -   name: set
+            set:
+                vadesecure.campaign.actions: '{{parse_json.message.actions}}'
+                vadesecure.campaign.id: '{{parse_json.message.id}}'
+                vadesecure.campaign.nb_messages_remediated: '{{parse_json.message.nb_messages_remediated}}'
+                vadesecure.campaign.nb_messages_remediated_read: '{{parse_json.message.nb_messages_remediated_read}}'
+                vadesecure.campaign.nb_messages_remediated_unread: '{{parse_json.message.nb_messages_remediated_unread}}'
+        -   name: set
+            set:
+                vadesecure.status: '{{parse_json.message.status.status}}'
+                vadesecure.substatus: '{{parse_json.message.status.substatus}}'
+    generic_email:
+        actions:
+        -   name: set
+            set:
+                email.local_id: '{{parse_json.message.id}}'
+                email.message_id: '{{parse_json.message.message_id}}'
```

Comment:
2 fields for the same information ?

Reply:
It is not the same information :mag_right:

Reply:
:man_facepalming: